iModelJsNodeAddon/api_package/ts/src/test/index.ts

```diff
@@ -36,7 +36,23 @@ chaiuse(chaiAsPromised);
 
 Logger.initializeToConsole();
 Logger.setLevelDefault(LogLevel.Error);
-iModelJsNative.logger = Logger;
+iModelJsNative.logger = {
+  // note: using private Logger fields is temporary until the version of `@itwin/core-bentley` is updated

```

Comment:
Will you file an issue and ensure we address it ASAP?

Reply:
Filed an issue for tracking: https://github.com/iTwin/imodel-native/issues/649.

I don't think there's a need to hurry - we were using those private members in our native code for a while anyway. I was planning to fix this at some point when a new `itwinjs-core` nightly build, using this new addon, is released.